lib/plugins/aws/invokeLocal/index.js

```diff
@@ -10,6 +10,9 @@ const chalk = require('chalk');
 const stdin = require('get-stdin');
 const spawn = require('child_process').spawn;
 const inspect = require('util').inspect;
+const download = require('download');
+const jsZip = require('jszip');
+const mkdirp = require('mkdirp');
```

Comment:
Looks like some of the dependencies here need to be added to `package.json`.

In `jsZip` case it needs to be moved from `devDependencies` to `dependencies`. We could also check if `archiver` supports unzipping to save one dependency.

Reply:
I'll look into `archiver`, but it's not a new dep afaik... i think archiver uses it internally

Reply:
switched to `download` lib's built-in extraction code 🙂